### ISSUE TITLE
Client Accessibility Improvements

### ISSUE THREAD

**Issue by cjcarlton** [DemocracyDevelopers/colorado-rla#209]
I'll use Chrome lighthouse and the siteimprove plugin to fix up any accessibility issues that get flagged

**Comment by cjcarlton** [comment]
I ran Google Chrome Lighthouse (run from developer tools with 'Accessibility' checked. As well as the SiteImprove plugin ( https://chromewebstore.google.com/detail/siteimprove-accessibility/djcglbmbegflehmbfleechkjhmedcopn ) with Level A, Level AA, and Level S Conformance selected.  I prioritized the county side since more people will be using it. There were a couple tables and things on the stateadmin side that I left since I didn't want to mess with their functionality.

Accessibility Improvements include:
-Fixing heading tags (h1-h4) so that every page starts with an h1 and they increment correctly
-Adding missing label tags (and sometimes aria-label) so screen readers can label inputs
-Color contrast fixes (made links in the app bold and underlined so their color contrast passes, made some button text black for green and orange buttons (white text was failing the check)
-Added a hidden 'skip to main content' link at the top of the page. You can see it if you tab over to it. This lets screen readers skip past the nav bar and into the main page content.
-added lang='en' to the <html> tag
